oak/client/per_call_policy.cc

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2019 The Project Oak Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "oak/client/per_call_policy.h"
+
+#include <map>
+#include <utility>
+
+#include "asylo/util/logging.h"
+#include "oak/common/nonce_generator.h"
+#include "oak/common/policy.h"
+
+namespace oak {
+
+namespace {
+constexpr size_t kPerCallNonceSize = 32;
+}
```

Comment:
why a different namespace?

Reply:
I don't want to expose this constant everywhere, just use it in this file, so I created a nested anonymous namespace.